Content.Shared/Blocking/BlockingSystem.User.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Content.Shared.Blocking;
 
-public sealed partial class BlockingSystem
+public abstract partial class SharedBlockingSystem
```

Comment:
```suggestion
public sealed partial class BlockingSystem
```
Revert this back to BlockingSystem